bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tmva/_gnn.py

```diff
@@ -0,0 +1,259 @@
+# Authors:
+# * Sanjiban Sengupta 01/2023
+# * Lorenzo Moneta    01/2023
+
+################################################################################
+# Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+from .. import pythonization
+from cppyy import gbl as gbl_namespace
+
+def getActivationFunction(model):
+    function = model._activation.__name__
+    if function == 'relu':
+        return gbl_namespace.TMVA.Experimental.SOFIE.Activation.RELU
+    else:
+        return gbl_namespace.TMVA.Experimental.SOFIE.Activation.Invalid
+
+def make_mlp_model(gin, model, target, type):
```

Comment:
Include docstrings of this function and document the input arguments. Same as other comments above

Reply:
made the change, thanks!